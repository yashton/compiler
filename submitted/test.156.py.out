(PUNCT "(")
(ID "N")
(PUNCT "=")
(NEWLINE)
(INDENT)
(LIT 5)
(ERROR "Illegal character '"

 A=6' on line 2)
