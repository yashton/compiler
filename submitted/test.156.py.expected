(PUNCT "(")
(ID "N")
(PUNCT "=")
(LIT 5)
(ERROR "newline in string")
